bundles/org.openhab.core.thing/src/main/java/org/openhab/core/thing/DefaultSystemChannelTypeProvider.java

```diff
@@ -173,13 +173,23 @@
             .withDescription("Controls the color of the light").withCategory("ColorLight").build();
 
     /**
-     * Color-temperature: default system wide {@link ChannelType} which allows changing the color temperature
+     * Color-temperature: default system wide {@link ChannelType} which allows changing the color temperature in percent
      */
     public static final ChannelType SYSTEM_COLOR_TEMPERATURE = ChannelTypeBuilder
             .state(new ChannelTypeUID(BINDING_ID, "color-temperature"), "Color Temperature", "Dimmer")
-            .withDescription("Controls the color temperature of the light").withCategory("ColorLight")
-            .withStateDescriptionFragment(StateDescriptionFragmentBuilder.create().withMinimum(BigDecimal.ZERO)
-                    .withMaximum(new BigDecimal(100)).withPattern("%d").build())
+            .withDescription("Controls the color temperature of the light from 0 (cold) to 100 (warm)")
+            .withCategory("ColorLight").withStateDescriptionFragment(StateDescriptionFragmentBuilder.create()
+                    .withMinimum(BigDecimal.ZERO).withMaximum(new BigDecimal(100)).withPattern("%d").build())
+            .build();
+
+    /**
+     * Color-temperature: default system wide {@link ChannelType} which allows changing the color temperature in Kelvin
+     */
+    public static final ChannelType SYSTEM_COLOR_TEMPERATURE_ABS = ChannelTypeBuilder
+            .state(new ChannelTypeUID(BINDING_ID, "color-temperature-abs"), "Color Temperature", "Number")
+            .withDescription("Controls the color temperature of the light in Kelvin").withCategory("ColorLight")
+            .withStateDescriptionFragment(StateDescriptionFragmentBuilder.create().withMinimum(BigDecimal.ONE)
+                    .withMaximum(new BigDecimal(1000000)).withPattern("%.0f K").build())
```

Comment:
Does 0-1.000.000 really make sense here?
Looking at https://www.lightingdesignlab.com/resources/articles/articles-lighting-fundamentals/color-temperature, I'd think a range from 1.000-10.000 might make sense, especially in case people want to attach a selection box or slider to it?

Reply:
1-1000000 is the possible range for it. But the visible spectrum for humans is from 1000-10000. I did not want to add to strict limits but it makes sense to only use what is most useful wrt to home automation.